pkg/oidc/oidc.go

```diff
@@ -95,6 +95,13 @@ type VerifierConfig struct {
 	oidc.Config
 }
 
+// String returns the string representation of the VerifierConfig.
+// It's used for logging purposes.
+func (config *VerifierConfig) String() string {
+	return fmt.Sprintf("ClientID: %s, SkipClientIDCheck: %t, SkipExpiryCheck: %t, SkipIssuerCheck: %t, InsecureSkipSignatureCheck: %t, SupportedSigningAlgs: %v, Now: %T",
+		config.ClientID, config.SkipClientIDCheck, config.SkipExpiryCheck, config.SkipIssuerCheck, config.InsecureSkipSignatureCheck, config.SupportedSigningAlgs, config.Now)
```

Comment:
config.Now type is `func() time.Time` and this information will be printed with %T.

Please adjust this field to show the value or remove this.

Reply:
"%T" is used on purpose. It will print the function signature if it's provided, otherwise it will print nil, which means no custom function provided. Field config.Now does not hold the time, it holds the function to call when the now time is needed.